### PR TITLE
Updated Energy Device model supporting on/off and power graphing 

### PR DESCRIPTION
**Pull request by NorthernMan54**
First draft of support for adding on/off status graphing to a energy device.

Discussion is part of #97

### CODE REVIEW

**Comment by NorthernMan54** [comment]
I’m going to refactor my change, and expose it as a energy2 device, and not change the existing energy service.  Mostly to avoid having to regression test the change, and allow a little more flexibility around the interface.

File: fakegato-history.js [hunk 1]
```diff
@@ -601,6 +655,19 @@ module.exports = function (pHomebridge) {
...
 					break;
+				case TYPE_WEATHER2:
+					if (!this.disableTimer)
+						if(entry.temp !== undefined) {	// allow on / to be added to the data stream
+							homebridge.globalFakeGatoTimer.addData({ entry: entry, service: this });
+						} else {
+							this._addEntry({ time: entry.time, status: entry.status });
+						}
+					else
+						if(entry.power !== undefined) {	// allow on / to be added to the data stream
```
Comment (thkl) [inline fakegato-history.js]: are u sure, that you have to check the power property on an TYPE_WEATHER2 entry ?

Comment (NorthernMan54) [reply fakegato-history.js]: You are correct, I should be checking the 'temp' field. Will fix in the next push

Tks for reviewing

**Comment by NorthernMan54** [comment]
Tks

**Comment by NorthernMan54** [comment]
Just published homebridge-tasmota v0.0.68 with support for custom devices.  As fakegato is a new feature, not sure what feedback I'm going to get.

**Pull request merged**